src/components/VisualFeedback/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { ResizeObserver } from 'resize-observer';
+
+import { UiMode } from '../ThemeContext';
+import { useThemeClassName } from '../../hooks';
+import styles from './styles.css';
+
+export interface VisualFeedbackProps {
+    className?: string;
+    disabled?: boolean;
+    uiMode?: UiMode;
+}
+
+function VisualFeedback(props: VisualFeedbackProps) {
+    const {
+        className,
+        disabled,
+        uiMode,
+    } = props;
+    const ref = React.useRef<HTMLDivElement>(null);
+
+    const handleResize = React.useCallback(() => {
+        if (ref.current) {
+            const { current } = ref;
+            const { parentElement } = current;
+
+            if (parentElement) {
+                const bcr = parentElement.getBoundingClientRect();
+                current.style.left = `${bcr.x}px`;
+                current.style.top = `${bcr.y}px`;
+                current.style.width = `${bcr.width}px`;
+                current.style.height = `${bcr.height}px`;
+            }
+        }
+    }, []);
+
+    React.useEffect(handleResize, [handleResize]);
```

Comment:
The useEffect on hadleResize doesn't make sense here.
Do you want to handle resize once on mount?

Reply:
should be `React.useEffect(handleResize, []);`